### ISSUE TITLE
New feature

### ISSUE THREAD

**Issue by Kwisher** [hamishcunningham/pi-tronics#108]
Is there a way to add fields to calculate daily, monthly & yearly totals?

**Comment by hamishcunningham** [comment]
erm, totals of what?! tnx!

**Comment by Kwisher** [comment]
Rainfall totals.

**Comment by Kwisher** [comment]
Also, anyway to limit the write to log.csv to only when the switch sends a signal? How about creating a separate date field (yyyy-mm-dd) & time field?

Thanks.

**Comment by petaflot** [comment]
> erm, totals of what?! tnx!

Yes, that happens when you put all your "unrelated" projects in a single git repo.

**Comment by petaflot** [comment]
the correct way to do this, in case libraries are used, is to use submodules:

https://git-scm.com/book/en/v2/Git-Tools-Submodules
